mathesar/models.py

```diff
@@ -45,14 +46,23 @@ def get_queryset(self):
         return super().get_queryset()
 
 
-class DatabaseObject(BaseModel):
-    oid = models.IntegerField()
+class ReflectingObject(BaseModel):
```

Comment:
I would call this `DatabaseObject` and the current `DatabaseObject` something like `ObjectWithOID`.

Reply:
I was planning to go with that first and then realised the Database model would be using it too and using DatabaseObject as a base model will be confusing, will come back to this.

Reply:
Mixins should be a better fit. Have to check if Django plays well with mixins that add fields

Reply:
Makes sense. I think `ReflectedObject` would be a bit clearer than `ReflectingObject`.